### DISCUSSION TITLE
Both svc split and svc  sd can't be used

### DISCUSSION THREAD

**Discussion by wx40217** [voicepaw/so-vits-svc-fork#221] [Q&A] [answered]
Hey I just download the code and try to split my dataset.But when I try this ,it always says "Error: No such command 'split'".However,I can still use the command to train such as "svc pre-resample".Did I miss something?
![image](https://user-images.githubusercontent.com/24869295/229570991-8f5260fa-8686-44b7-8b85-180cb90cd52a.png)

**Comment by sbersier** [comment, marked as answer]
I think the correct command is: `svc pre-split`
